tests/unit/test_cli.py

```diff
@@ -33,3 +33,22 @@ def test_skip_no_ucx(caplog, mocker):
     mocker.patch("databricks.labs.ucx.installer.InstallationManager.for_user", return_value=None)
     skip(schema="schema", table="table")
     assert [rec.message for rec in caplog.records if "UCX configuration" in rec.message]
+
+
+def test_revert(mocker):
+    mocker.patch("databricks.sdk.WorkspaceClient.__init__", return_value=None)
+    current_user = MagicMock()
+    current_user.me.return_value = User(user_name="foo", groups=[ComplexValue(display="admins")])
+    current_user.return_value = None
+    mocker.patch("databricks.sdk.WorkspaceClient.current_user", return_value=current_user)
+    mocker.patch("databricks.labs.ucx.installer.InstallationManager.__init__", return_value=None)
+    installation = MagicMock()
+    installation.config.warehouse_id = 123
+    mocker.patch("databricks.labs.ucx.installer.InstallationManager.for_user", return_value=installation)
+    mocker.patch("databricks.labs.ucx.framework.crawlers.StatementExecutionBackend.__init__", return_value=None)
+    mocker.patch("databricks.labs.ucx.hive_metastore.tables.TablesMigrate.__init__", return_value=None)
+    rmt = mocker.patch(
+        "databricks.labs.ucx.hive_metastore.tables.TablesMigrate.revert_migrated_tables", return_value=None
+    )
```

Comment:
```suggestion
tc = create_autospec(TablesCrawler)
ws = create_autospec(WorkspaceClient)
sql_backend = create_autospec(SqlBackend)
# mock executions of methods from tc,ws,..., DO NOT monkey patch the existing methods.
tm = TablesMigrate(tc, ws, sql_backend)

# assert calls to methods of tc, ws, sql_backend later
```

this is malpractice. don't do it.

Reply:
I was trying to test the actual CLI. As I see nobody does that, I removed the test.